test/unit/helpers/ast-node-info-test.js

```diff
@@ -0,0 +1,13 @@
+var assert = require('power-assert');
+var parse = require('htmlbars/dist/cjs/htmlbars-syntax').parse;
+var AstNodeInfo = require('../../../lib/helpers/ast-node-info');
+
+describe('isImgElement', function() {
+  it('can detect an image tag', function() {
+    var tableAst = parse('<table></table>');
+    assert(AstNodeInfo.isImgElement(tableAst.body[0]) === false);
+
+    var imgAst = parse('<img />');
```

Comment:
Interesting. Does HTMLBars throw an error for a closing `/`? It's optional and that's how I've always written my `img` tags. 

> Optionally, a "/" character, which may be present only if the element is a void element.

https://www.w3.org/TR/html-markup/syntax.html#void-element

(also just learned all about void elements, woo! 🎉 )